main.go

```diff
@@ -619,7 +623,7 @@ func main() {
 		log.Println("server started")
 		for running {
 			time.Sleep(1 * time.Second)
-			for uint64(settings.MaxCacheSizeInMebibytes)*1024 < diskUsed {
+			for uint64(settings.MaxCacheSizeInMebibytes) < diskUsed {
```

Comment:
diskUsed is in bytes I'm pretty sure so I'm not sure this will work as intended

Reply:
Well yes and with your new loading of settings MaxCacheSizeInMebibytes is now in bytes too.